Fabric.Identity.API/scripts/tests/CatalystDosIdentity.functional.tests.ps1

```diff
@@ -63,6 +60,19 @@ Describe 'Identity Cli Functional Tests' {
                     $error | Should -Be '{"error":"invalid_client"}'
                 }
             }
+
+            It 'Should return an exception when the request does not have a scope' {
+                try {
+                    Get-AccessToken -identityUrl $identityUrl -clientId "fabric-installer" -secret $installerSecret
```

Comment:
I think this should still be a valid request here (no scope).  I think this test should probably create a throwaway client to test with with valid scopes such as "fabric/identity.manageresources" or similar, and then try and get a token on that client.

Reply:
I don't follow. No scopes doesn't work when trying to get an access token